src/iop/filmicrgb.c

```diff
@@ -1105,19 +2184,29 @@ void commit_params(dt_iop_module_t *self, dt_iop_params_t *p1, dt_dev_pixelpipe_
   dt_iop_filmicrgb_params_t *p = (dt_iop_filmicrgb_params_t *)p1;
   dt_iop_filmicrgb_data_t *d = (dt_iop_filmicrgb_data_t *)piece->data;
 
-  d->preserve_color = p->preserve_color;
+  // source and display greys
+  float grey_source, grey_display;
```

Comment:
= 0.0f for both.

Reply:
initialized with 0.18 and 0.46 instead, but same result.